dist/profile/templates/confluence/vhost.conf

```
@@ -3049,6 +3049,7 @@ RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Features\+controlled\+by\+system\+properties$" "https://jenkins.io/doc/book/managing/system-properties/" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Groovy\+Hook\+Script$" "https://jenkins.io/doc/book/managing/groovy-hook-scripts/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/User\+Content$" "https://www.jenkins.io/doc/book/managing/user-content/" [NC,L,QSA,R=301]
```

Would you mean 
```suggestion
RewriteRule "^/display/JENKINS/User\+Content$" "https://www.jenkins.io/doc/book/managing/users/" [NC,L,QSA,R=301]
```

Because the url you set redirect to a 404 error

---

No, the url is ok, there is a pending PR for the new doc https://github.com/jenkins-infra/jenkins.io/pull/3896

---

https://github.com/jenkins-infra/jenkins.io/pull/3896 has now been merged